ts/features/itwallet/walletInstance/store/reducers/index.ts

```diff
@@ -41,6 +50,19 @@ const migrations: MigrationManifest = {
       ...prevState,
       status: prevState.status ? pot.some(prevState.status) : pot.none
     };
+  },
+  // Move the old Wallet Attestation into the new structure
+  "1": state => {
+    const prevState = state as PersistedState & {
+      attestation: string | undefined;
+      status: pot.Pot<WalletInstanceStatus, NetworkError>;
+    };
+    return {
+      ...prevState,
+      attestation: {
+        jwt: prevState.attestation
+      }
+    };
```

Comment:
Migrations should be decoupled from the store structure. What do you think?

```suggestion
  "1": state => {
    return {
      ...state,
      attestation: {
        jwt: state.attestation
      }
    };
```

Reply:
I removed the store types in 0458729c31b740d71df9411e5671041af3708109.